### PR TITLE
Feature/scanner

### PR DESCRIPTION
**Pull request by Baughn**
This acts basically like a redstone comparator, except when it doesn't. See the tooltip in Scanner.kt.

### CODE REVIEW

**Comment by AMIDIBOSS** [comment]
Did you just... 
Remove my model?

**Comment by Baughn** [comment]
Um, no?

I moved it.

File: src/main/java/mods/eln/sixnode/ElectricalFuse.kt [hunk 1]
```diff
@@ -205,11 +205,7 @@ class ElectricalFuseHolderElement(sixNode: SixNode, side: Direction, descriptor:
     }
 
     override fun onBlockActivated(entityPlayer: EntityPlayer?, side: Direction?, vx: Float, vy: Float, vz: Float): Boolean {
-        if (Utils.isPlayerUsingWrench(entityPlayer)) {
-            front = front.nextClockwise
-            sixNode.reconnect()
-            return true
-        }
+        if (super.onBlockActivated(entityPlayer, side, vx, vy, vz)) return true
```
Comment (cm0x4D) [inline src/main/java/mods/eln/sixnode/ElectricalFuse.kt]: Maybe 
```java
if (onBlockActivatedRotate(entityPlayer)) return true
```
would be more clear when actually reading the code (multiple locations).

**Review by cm0x4D — COMMENTED** [review]
Tested it and it works fine ^^.

-The texture is a bit boring. Maybe the side that actually does the scanning should have a different color.

- Maybe a mode switch could be added to the model in order to see without WAILA or the multimeter which mode is active.

File: src/main/java/mods/eln/sixnode/ElectricalFuse.kt (hunk 1 above)
Comment (Baughn) [reply src/main/java/mods/eln/sixnode/ElectricalFuse.kt]: Concur.

**Comment by AMIDIBOSS** [comment]
@cm0x4D Good Idea

**Review by cm0x4D — APPROVED** [review]

**Review by AMIDIBOSS — COMMENTED** [review]
And what that export script is for?

**Comment by cm0x4D** [comment]
Attention: sixnode models and transparentnode models do not have the same settings when exported to wavefront files. You may need to check and adapt the rendering code. Imho it would be great to have one setting for all models, even if this means we have to adapt the rendering code.

**Comment by Baughn** [comment]
The transparentnode models *also* don't have all the same settings, because I didn't find it documented anywhere. Yeah, long-term that's the plan. I'll probably do it around the time we're porting this to 1.10, since then I'll need to redo a lot of the rendering anyway.

For now, I just didn't want to lose the script.

**Comment by Baughn** [comment]
...and also, there are some optimizations I tend to do on models; e.g. joining every object that can be joined, so that only a single draw call is needed. If I can move that into the *script*... which I can, of course.

It'll be great. You'll see. ;)

**Comment by Baughn** [comment]
I think this is ready now. Any final comments?

**Comment by cm0x4D** [comment]
On my machine it crashes on startup: http://pastebin.com/usFYvcxB

I suspect the run of the blender script as a problem...

**Comment by Baughn** [comment]
Interesting.

I've un-run it. How do things look now?

**Comment by cm0x4D** [comment]
My GIT client is now in nirvana. Cannot do anything anymore...

**Comment by cm0x4D** [comment]
That was not funny. I had to merge in order to stash in order to checkout another branch in order to delete and re-create and checkout this branch. On the bright side, now it does not crash anymore 😄

**Comment by cm0x4D** [comment]
Everything works for me now.

@AMIDIBOSS: Could you add indicators for the modes next to the LEDs, would make it clearer.

**Pull request merged**